showcase/app/controllers/page-components/pagination/index.ts

```diff
@@ -8,22 +8,39 @@ import { action } from '@ember/object';
 import { tracked } from '@glimmer/tracking';
 import { inject as service } from '@ember/service';
 
+import type RouterService from '@ember/routing/router-service';
+
+import type { HdsPaginationDirections } from '@hashicorp/design-system-components/components/hds/pagination/types';
+import type { HdsTableThSortOrder } from '@hashicorp/design-system-components/components/hds/table/types';
+
+import type { PageComponentsPaginationModel } from 'showcase/routes/page-components/pagination/index';
+
 // uncomment this to override the `atob/btoa` functions for debugging
 // const atob = (s) => s;
 // const btoa = (s) => s;
```

Comment:
```suggestion
// const btoa = (s: string) => s;
```
[Nit] if this does get commented out it will cause type errors